### DISCUSSION TITLE
Pas d'import cotes Tennis betclic

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#27] [General]
Bonjour,
Pour info avec  l'interface graphique impossible de scroll les cotes Tennis Betclic : 

![image](https://user-images.githubusercontent.com/78261069/109928308-9d09d180-7cc5-11eb-8582-80023db1ac2c.png)

![image](https://user-images.githubusercontent.com/78261069/109928368-ac891a80-7cc5-11eb-86da-d08eeb251604.png)

Cela tourne sans cesse sans se terminer.

Merci par avance

Cordialement,

**Comment by pretrehr** [comment]
Bonjour
Je vous invite à récupérer la dernière version du projet, vous n'aurez plus ce problème, et la récupération de cotes sera beaucoup plus rapide

**Reply by kreon76** [reply]
Merci beaucoup...
J'ai fait ce que vous m'aviez expliqué. Je me suis placé dans mon répertoire C:\Python\Sportbetting
Avec Git j'ai fait : git reset --hard puis git pull
Et maintenant qd je lance l'interface elle ne se lance plus :(

![image](https://user-images.githubusercontent.com/78261069/109931975-e0663f00-7cc9-11eb-9d82-30bf7cc46f68.png)

**Reply by pretrehr** [reply]
Ah oui, pour corriger ce problème vous avez simplement à faire un
```
pip install -r requirements.txt
```

Je n'y avais pas pensé la dernière fois mais c'est toujours une bonne idée de faire cette commande après un
```
git pull
```

**Reply by kreon76** [reply]
parfait c'est reparti !!! Grand Merci !
